src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="home-page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="home-page__content">
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="IMac"
+        class="card__img"
+      >
+      <div class="card__name">
+        <span>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</span>
+      </div>
+      <div class="card__code">
+        <span>Product code: 195434</span>
+      </div>
+      <div class="card__rate">
```

Comment:
i guess if u put stars block and reviews element in flex element, you will be able to position them more accurate due to mockup

Reply:
agree